### PR TITLE
Support right clicking the database to start the importer task

### PR DESCRIPTION
**Pull request by aaomidi**

### CODE REVIEW

File: extensions/import/src/wizard/pages/fileConfigPage.ts [hunk 1]
```diff
@@ -55,13 +55,11 @@ export class FileConfigPage extends ImportPage {
 	}
 
 	async onPageEnter(): Promise<boolean> {
-		await this.populateServerDropdown();
-		await this.populateDatabaseDropdown();
-		await this.populateSchemaDropdown();
-		return true;
+		return await this.populateServerDropdown().then(() => this.populateDatabaseDropdown()).then(() => this.populateSchemaDropdown());
```
Comment (llali) [inline extensions/import/src/wizard/pages/fileConfigPage.ts]: why did you change await to then?

Comment (aaomidi) [reply extensions/import/src/wizard/pages/fileConfigPage.ts]: Readability was the only reason

Comment (llali) [reply extensions/import/src/wizard/pages/fileConfigPage.ts]: well I actually think the other one was more readable but that's my opinion ...

---
In reply to: [210407076](https://github.com/Microsoft/sqlopsstudio/pull/2233#discussion_r210407076) [](ancestors = 210407076)

Comment (aaomidi) [reply extensions/import/src/wizard/pages/fileConfigPage.ts]: Oh I can change it back! Don't mind either way :)
